src/bower_components/emby-webcomponents/homesections/homesections.js

```diff
@@ -273,37 +273,16 @@
     }
 
     function loadlibraryButtons(elem, apiClient, user, userSettings, userViews) {
-
-        return Promise.all([getAppInfo(apiClient), getDownloadsSectionHtml(apiClient, user, userSettings)]).then(function (responses) {
-
-            var infoHtml = responses[0];
-            var downloadsHtml = responses[1];
-
+        return new Promise(function (resolve, reject) {
```

Comment:
Hmm, there's nothing to resolve these new promises unless I'm missing something.

Reply:
Yeah, turns out I had it right the first time. The promises were only made to call some async methods and they were resolved right here. Without the method calls I removed the promises aren't necessary, I'll remove them in a minute.

Reply:
The way I see it there's two solutions. Removing the promises and not adding them to the promise array (this one you missed), or resolving them immediately. I prefer the former. Sorry I didn't suggest that instead...

Reply:
Isn't that what I did the first time though? I removed the promises and didn't return any values from those two methods.

Reply:
Check the current source for my original solution, I added it again.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/37/files#diff-953f15eac453b39554bd82baac54837eR151

Reply:
ahhhh, okay. I'll fix it properly now haha

Reply:
How about my latest commit? Removed the return statement so it just loads normally.